app/managers/session_manager.ts

```diff
@@ -168,6 +168,8 @@ class SessionManager {
 
         await this.terminateSession(serverUrl, removeServer);
 
+        await removePushDisabledInServerAcknowledged(extractCleanDomain(serverUrl));
```

Comment:
Why do we need to `extractCleanDomain`. As far as I know, everywhere else we use directly the `serverUrl`, right?

Reply:
I'd say we don't need to remove it on logout, it should remain there across logouts.. but it should be removed if the server gets configured to use push notifications correctly in the future.